amity/amity.py

```diff
@@ -0,0 +1,751 @@
+import sqlite3
+import random
+import os
+
+from termcolor import cprint, colored
+
+from amity.database.database_file import Database
+
+
+class Amity(object):
+    all_persons = {
+        'staff': [],
+        'fellow': []
+        }
+    all_allocations = {
+        'office': {},
+        'living_space': {}
+        }
+    all_unallocated = []
+    all_rooms = {
+        'living_space': [],
+        'office': []
+        }
+
+    def create_room(self, room_type, room_name):
+        """
+
+        Create a list of rooms depending on the user's input
+
+        """
+        if room_type == 'O':
```

Comment:
you can check for upper and lower case via: `if room_type.lower()` which should render the variable in lower case if upper